### ISSUE TITLE
Elixir Support

### ISSUE THREAD

**Issue by donifer** [apex/up#524]
[Elixir](https://elixir-lang.org) support

**Comment by lukeed** [comment]
I think this only makes sense to do once Fargate support is here. Elixir on Lambda doesn't really make sense from an Elixir point of view.

**Comment by tj** [comment]
Yeah sort of defeats the purpose, but the same could be said about Go I suppose

**Comment by lukeed** [comment]
Maybe 😆 but at least Go still shines with one process. Elixir was _made_ for handling hundreds at minimum.
